src/data/row.rs

```diff
@@ -25,6 +25,13 @@ pub enum RowError {
     ConflictOnEmptyRow,
 }
 
+#[derive(iter_enum::Iterator)]
+
+enum Columns<I1, I2> {
```

Comment:
Let's just remove this blank line, then all looks good!

Reply:
Uh.. why did this go in? will do fixup